picoscope/picobase.py

```diff
@@ -335,6 +335,21 @@ def getMaxMemorySegments(self):
         segments = self._lowLevelGetMaxSegments()
         return segments
 
+    def setExtTriggerRange(self, VRange=0.5):
+        """ This function sets the range for the EXT trigger channel
+            VRange MUST either be 0.5 or 5 for +/- 500mV or 5V ranges
+            respectively
+            This is only implemented for PS4000 series devices.
+        """
+        if VRange == 0.5:
```

Comment:
Why not use the code in https://github.com/colinoflynn/pico-python/blob/master/picoscope/picobase.py#L228

It would actually work for all picoscopes too.

Reply:
I didn't use that code because I don't like it :-) and because it doesn't seem to fit for the simple case here where only two non-consecutive API integer range values are valid for the shared object/dll function call.

Maybe using a dict would be better here?
```
EXT_RANGE = {"5V": 8, "0.5V": 5}
```

Reply:
Unless they specify that they use different APIi codes, I don't think it is necessary.

The worst that would happen is that it gets caught in the lowLevel error checking that you already did in [Line 403](https://github.com/colinoflynn/pico-python/pull/117/files#diff-183381102b9c250dd4d1d4527efabdf8R403)

Reply:
What you proposed is fine, I don't know why we picked a different model:
https://github.com/colinoflynn/pico-python/blob/master/picoscope/picobase.py#L84
but maybe stick to that for consistency?
I think we wanted to print a string in scientific notation and that was what we thought of.

You can leave a note about using newer python constructs if you feel like it. Colin was using python2.7 at the time and python 3 just wasn't ready for prime time.